src/transformers/pipelines/audio_utils.py

```diff
@@ -70,6 +75,10 @@ def ffmpeg_microphone(
             The indentifier of the input device to be used by ffmpeg (i.e. ffmpeg's '-i' argument). If unset,
             the default input device will be used. See `https://www.ffmpeg.org/ffmpeg-devices.html#Input-Devices`
             for how to specify and list input devices.
+        ffmpeg_additional_args (`list`, *optional*):
```

Comment:
better to specify what should type of objects can contained in the list 

```suggestion
        ffmpeg_additional_args (`List[str]`, *optional*):
```